packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -48,13 +76,9 @@ const onRenderParticipantDefault = (
     }
   }
 
-  const menuItems: IContextualMenuItem[] = [];
-  if (participant.userId !== myUserId && onParticipantRemove) {
-    menuItems.push({
-      key: 'Remove',
-      text: 'Remove',
-      onClick: () => onParticipantRemove(participant.userId)
-    });
+  let defaultMenuItems = defaultOnFetchParticipantMenuItems(participant.userId, myUserId, onParticipantRemove);
+  if (onFetchParticipantMenuItems) {
+    defaultMenuItems = onFetchParticipantMenuItems(participant.userId, myUserId, defaultMenuItems);
```

Comment:
nit: we can't say it's the default MenuItem list if we are fetching from Contoso customization

Reply:
would changing the name to `menuItems` make it more clear?

Reply:
I like just `menuItems`